packages/javascript/bh-shared-ui/src/components/EntityInfo/EntitySelectorsInformation.tsx

```
@@ -0,0 +1,147 @@
+            setExploreParams({
+                expandedPanelSections: ['Selectors'],
+            });
+        } else {
+            setExploreParams({
+                expandedPanelSections: [],
+            });
+        }
+    };
+
+    const memberInfoQuery = useQuery(['asset-group-member-info'], () => {
+        return apiClient.getAssetGroupTagMemberInfo(tagId!!, memberId!!).then((res) => {
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)